pyproject.toml

```diff
@@ -2,15 +2,21 @@
 name = "horreum"
 version = "0.13-dev"
 description = "Horreum python library"
+keywords = ["horreum", "performance", "change-detection"]
 authors = ["Andrea Lamparelli <alampare@redhat.com>"]
+maintainers = ["Andrea Lamparelli <alampare@redhat.com>"]
```

Comment:
What should I add here as `author` and `maintainers`?

Reply:
Hm, I think you can add john and myself I think as maintainers. We'll see how it goes with this repo and how much changes we'll need in the future.

Reply:
Done :heavy_check_mark: